retroshare-gui/src/gui/feeds/GxsChannelPostItem.cpp

```diff
@@ -83,7 +83,7 @@ GxsChannelPostItem::GxsChannelPostItem(FeedHolder *feedHolder, uint32_t feedId,
 
 	setMessageVersions(v) ;
     setup();
-
+    mLoaded = false;
     loadGroup();
```

Comment:
Only need in /retroshare-gui/src/gui/feeds/GxsChannelPostItem.cpp:176
void GxsChannelPostItem::setup()

Reply:
you mean add after line 176 mLoaded = false;

Reply:
Fixed by PR #1909